core/startos/src/system.rs

```diff
@@ -871,6 +877,57 @@ pub async fn clear_system_smtp(ctx: RpcContext) -> Result<(), Error> {
     }
     Ok(())
 }
+pub async fn test_system_smtp(
+    _: RpcContext,
+    SmtpValue {
+        server,
+        port,
+        from,
+        login,
+        password,
+    }: SmtpValue,
+) -> Result<(), Error> {
+    use rustls_pki_types::pem::PemObject;
+
+    let Some(pass_val) = password else {
+        return Err(Error::new(
+            eyre!("mail-send requires a password"),
+            ErrorKind::InvalidRequest,
+        ));
+    };
+
+    let mut root_cert_store = RootCertStore::empty();
+    let pem = tokio::fs::read("/etc/ssl/certs/ca-certificates.crt").await?;
+    for cert in CertificateDer::pem_slice_iter(&pem) {
+        root_cert_store.add_parsable_certificates([cert.with_kind(ErrorKind::OpenSsl)?]);
+    }
+    let crypto_provider = rustls::crypto::ring::default_provider();
+    let _ = CryptoProvider::install_default(crypto_provider.clone());
```

Comment:
```suggestion
    let _ = CryptoProvider::install_default(crypto_provider.clone());
```
does it not work without this? I would really like to avoid doing this